siliconcompiler/core.py

```diff
@@ -989,7 +989,7 @@ def _find_sc_file(self, filename, missing_ok=False, search_paths=None):
                 scpaths.extend(os.environ['SCPATH'].split(os.pathsep))
             scpaths.append(self.scroot)
 
-        searchdirs = ', '.join(scpaths)
+        searchdirs = ', '.join([str(p) for p in scpaths])
```

Comment:
Does this really make a difference?
I would have assumed it gets casted automatically.

Reply:
They don't appear to be cast to str automatically.